cardano-constitution/cardano-constitution.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-constitution
-version:            1.32.0.0
+version:            1.34.0.0
```

Comment:
Did we skip 1.33 last release (I wonder why) ?

Reply:
It's not released on chap atm, so it is fine

Reply:
Also the prepare-release.sh by design will not update it, which is as expected